app/src/editor/canvas/components/ModuleSearch.jsx

```diff
@@ -1,17 +1,52 @@
+// @flow
+
 import React from 'react'
+import { connect } from 'react-redux'
 import startCase from 'lodash/startCase'
 
+import type { Stream } from '$shared/flowtype/stream-types'
+import type { Filter } from '$userpages/flowtype/common-types'
+
+import { getStreams, updateFilter } from '$userpages/modules/userPageStreams/actions'
+import { selectStreams } from '$userpages/modules/userPageStreams/selectors'
```

Comment:
`userPageStreams` could be shared module, though I wonder if it would be better to leave out `redux` altogether from the editor and use only the API request services from `userPageStreams`. @timoxley can probably weigh on this more.

Reply:
> I wonder if it would be better to leave out redux altogether from the editor and use only the API request services from `userPageStreams`

Yeah I agree. Let the components manage their own data.

Reply:
Yeah, I'll refactor this 👍